README.md

```diff
@@ -89,6 +89,32 @@ To run the hook over multiple files at the same time:
       files: ^requirements-dev\.(in|txt)$
 ```
 
+To synchronize your dependencies upon branch checkout, pull or rebase:
+```yaml
+default_install_hook_types:
+  - pre-commit
+  - post-checkout
+  - post-merge
+  - post-rewrite
+repos:
+  - repo: https://github.com/astral-sh/uv-pre-commit
+    # uv version.
+    rev: 0.5.9
+    hooks:
+      - id: uv-sync
+```
+In order to install this hook, you either need to specify `default_install_hook_types` (as above) and run `pre-commit install`, 
```

Comment:
```suggestion
```

In order to install this hook, you either need to specify `default_install_hook_types` (as above) and run `pre-commit install`, 
```

Reply:
_sigh_ GitHub.

I tried to fix the spacing in the suggestions but alas. Can you use newlines for consistency with the rest of the document?